### ISSUE TITLE
[BUG] Not work with Claude API

### ISSUE THREAD

**Issue by jtsang4** [Bin-Huang/chatbox#915]
**Bug Description**
I'm using the ChatBox@0.6.7 with Claude Model, and meet the error `API Error: {"error":{"type":"invalid_request_error","message":"prompt must end with \"\n\nAssistant:\" turn"}}` after sending the first message.

**Steps to Reproduce**
Just send any message when using Claude Model.

**Desktop (please complete the following information):**

-   Operating System: MacOS
-   Application Version: 0.6.7

**Comment by marioseixas** [comment]
I've got the fix, just end every prompt of your with two linebreaks and 'Assistant:'

For example:

Prompt:
```
Who are you?


Assistant:
```

Answer:
```
I'm Claude, an AI assistant created by Anthropic.
```

**Comment by Bin-Huang** [comment]
Fixed in v0.6.8 https://github.com/Bin-Huang/chatbox/releases
